src/ralph/lib/transitions/models.py

```diff
@@ -101,8 +101,11 @@ def _get_history_dict(data, instance, runned_funcs):
                     value = str(field.rel.to.objects.get(pk=v))
             except FieldDoesNotExist:
                 field = func.form_fields[k]['field']
-                if isinstance(field, forms.ChoiceField):
+                if type(field) == forms.ChoiceField:
```

Comment:
i'd prefer not to use `type` comparison. you could swap these if-else to first check `if isinstance(field, forms.ModelMultipleChoiceField):`, then `elif isinstance(field, forms.ChoiceField):`